docs/continuous_batching.md

```diff
@@ -0,0 +1,159 @@
+            "Tell me an interesting fact about llamas.",
+            "What is the best way to cook a steak?",
+            "Are you familiar with the Special Theory of Relativity and can you explain it to me?",
+            "Recommend some interesting books to read.",
+            "What is the best way to learn a new language?",
+            ]
+
+tokenizer = AutoTokenizer.from_pretrained(model_name, trust_remote_code=True, padding_side="left")
+# if the tokenizer has no pad_token, you can specify it.
+tokenizer.pad_token = tokenizer.eos_token
+pad_token_id = tokenizer.pad_token_id
+inputs = tokenizer(ps, padding=True, return_tensors='pt').input_ids
+
+model = Model()
+model.init(model_name, use_quant=True, weight_dtype="int4", compute_dtype="int8")
+# greedy search example, top_k_top_p sampling and beam_search also supported
+# do not forget to pass pad_token_id
+outputs = model.generate(inputs, max_new_tokens=128, do_sample=False, pad_token=pad_token_id)
+ans = tokenizer.batch_decode(outputs, skip_special_tokens=True, clean_up_tokenization_spaces=False)
+for a in ans:
+    print(a)
+    print("===========================")
+```
+> Note: Not every model supports multi-batching inference and most of them are under construction, please refer to [Supported Models](#supported-models).
+
+## Server
+We supply a corresponding [script](../scripts/python_api_example_for_model_server.py) for server usage.
+You can modify the `max_request_num` for setting the maximum bearable requests.
+
+>Note: 1.The server system we developed is just a prototype now. It may change interface and usage later. 2. Not every model supports server mode and most of them are under construction, please refer to [Supported Models](#supported-models).
+
+## Supported Models
+You can refer to [developer_document](../developer_document.md#22-inference-process) for adding continuous batching inference feature in your own customized model.
+<table>
+  <thead>
+    <tr>
+      <th>Models</th>
+      <th align="center">Continuous Batching Support</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>
+
+[LLaMA-7B](https://huggingface.co/decapoda-research/llama-7b-hf), [LLaMA-13B](https://huggingface.co/decapoda-research/llama-13b-hf), [LLaMA2-7B](https://huggingface.co/meta-llama/Llama-2-7b-chat-hf), [LLaMA2-13B](https://huggingface.co/meta-llama/Llama-2-13b-chat-hf), [LLaMA2-70B](https://huggingface.co/meta-llama/Llama-2-70b-chat-hf)</td>
+      <td align="center">✅</td>
+    </tr>
+    <tr>
+      <td>
+[CodeLlama-7b](https://huggingface.co/codellama/CodeLlama-7b-Instruct-hf)</td>
+      <td align="center">✅</td>
+    </tr>
+    <tr>
+      <td>
+[Solar-10.7B](https://huggingface.co/upstage/SOLAR-10.7B-Instruct-v1.0)</td>
+      <td align="center">✅</td>
+    </tr>
+    <tr>
+      <td>
+[Neural-Chat-7B-v3-1](https://huggingface.co/Intel/neural-chat-7b-v3-1), [Neural-Chat-7B-v3-2](https://huggingface.co/Intel/neural-chat-7b-v3-2)</td>
+      <td align="center">✅</td>
```

Comment:
could you also add [Magicoder-6.7B](https://huggingface.co/ise-uiuc/Magicoder-S-DS-6.7B)
here for support list  ?

Reply:
added in https://github.com/intel/neural-speed/pull/186/commits/b3066fc012946412ec5d5ee444a9508660fa8234